### ISSUE TITLE
User Story: AllSpice user wants to receive email and accept invite to org/team

### ISSUE THREAD

**Issue by allthedoll** [allthedoll/allspice-product#58]
## TL;DR

- As an extant AllSpice user I want to get an email to join a team or org so that I can join the org or team I'm invited to.

## Product requirements TL;DR

- [ ] User added via username receives email to join org/team
  - [ ] User can click accept link to join org/team
  - [ ] Website shows they are now a part of the org/team  

<img width="780" alt="Screenshot 2023-10-06 at 12 53 23 PM" src="https://github.com/allthedoll/allspice-product/assets/5758031/cef54254-44f3-44aa-bb23-cf43e93a9e98">

## Design requirements

- TBD

## Technical requirements

- TBD

## Timeframe

- Noted in [onboarding initiative](https://docs.google.com/document/d/1N5YY-I72F4WMXZt2NSJVvB2eSypoLuMATuKbO_rVuQs/edit?usp=sharing)
